### DISCUSSION TITLE
Limiter input

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#194] [Internal Design] [answered]
Hi Hantao,

I have a question about the ``Limiter``, the input ``u`` does not apply ``dummify`` like ``lower`` and ``upper``. Is it designed so?

Background: sometimes if we have an equation for the ``Limiter`` input, it can be redundant to put a new variable as the input, so I'm curious about such design.

Related codes are in ``discrete.py``
```python
...
        self.u = u
        self.lower = dummify(lower)
        self.upper = dummify(upper)
...
```

Regards,
Jinning

**Comment by cuihantao** [comment, marked as answer]
Dummy variables can only be used in equations later.

The input is not taken as an equation. Rather, the v field of u is directly accessed when comparing.

Regards,
Hantao Cui

**Reply by jinningwang** [reply]
Thanks for your answer!

Regards,
Jinning
